code/src/containers/forms/product-form/ProductForm.scss

```diff
@@ -0,0 +1,180 @@
+  }
+
+  &__image-section {
+    grid-row: 1 / 2;
+    grid-column: 1 / 2;
+  }
+
+  &__main-info-section {
+    grid-column: 2 / 3;
+    grid-row: 1 / 3;
+    display: flex;
+    flex-direction: column;
+    gap: spacing(5);
+  }
+
+  &__additional-info-section {
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
+
+    .product-form__body {
+      padding-bottom: spacing(3);
+    }
+  }
+
+  &__category-select {
+    padding: spacing(3) spacing(11) spacing(3) spacing(14) spacing(3);
+
+    &-label {
+      display: inline;
+    }
+  }
+  &__quantity-price-container {
+    display: flex;
+    gap: spacing(5);
+    margin-bottom: spacing(5);
+  }
+
+  &__text-input {
+    margin-bottom: spacing(4);
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__description-input {
+    padding: 1.75px 1px !important; // To rewrite mui styles
+  }
+
+  &__header-title {
+    font-size: $fs-base;
+    font-weight: bold;
+    color: var(--new-product-form-header-title-color, $white);
+  }
+
+  &__image-preview-text {
+    color: var(--spa-text-color, $gray);
+    text-align: center;
+  }
+
+  &__image {
+    @include box(160px);
+
+    object-fit: cover;
+  }
+
+  &__footer {
+    display: grid;
+    grid-template-columns: calc(30% - 10px) calc(70% - 10px);
+    grid-gap: spacing(5);
+
+    &-button {
+      border-radius: 3px;
+      height: 47px;
+    }
+  }
+
+  &__visibility-checkbox {
+    padding: spacing(1) spacing(3);
+
+    &-label {
+      color: var(
+        --new-rpoduct-form-checkbox-color,
+        $light-alpha
+      ) !important; // To rewrite mui styles and styles setted in AppCheckbox
+    }
+
+    & svg {
+      fill: var(
+        --new-rpoduct-form-checkbox-color,
+        $light-alpha
+      ) !important; // To rewrite mui styles and styles setted in AppCheckbox
+    }
+  }
+
+  &__category-select-container {
+    margin-bottom: spacing(4);
+  }
+
+  &__category-select-helper {
+    padding-left: spacing(3);
+  }
+
+  &__language-buttons {
+    margin-left: auto;
+    display: flex;
+    gap: spacing(2);
+  }
+
+  &__language-button {
+    border-radius: 5px;
+    padding: spacing(2) spacing(3);
+    background-color: transparent;
+    border-color: transparent;
+    color: var(--product-form-language-button-color, $gray-light);
+    outline: none !important; // to overwrite mui styles
+
+    &:hover {
+      color: var(--product-form-language-button-hover-color, $white);
+    }
+  }
+
+  &__language-button--error:not(.product-form__language-button--active) {
+    border: 1px solid var(--product-form-language-button-error-color, $red) !important; // to overwrite mui styles
+    color: var(--product-form-language-button-error-color, $red);
+  }
+
+  &__language-button--active {
+    color: $white;
+    background-color: var(
+      --product-form-language-button-active-color,
+      #636363
+    ) !important; // to overwrite mui styles
+    border-color: var(--product-form-language-button-active-color, #636363);
```

Comment:
Why this color is not inside our design system?

Reply:
I haven't found color that would suit this case inside our design system, so I added custom color. It is used only here, so I haven't added it globally. In latest commit I moved it to separate scss var inside of this file